example/offboard_velocity/offboard_velocity.cpp

```diff
@@ -76,12 +78,9 @@ bool offb_ctrl_ned(Device &device)
 
     // Let yaw settle.
```

Comment:
Don't think we need this note - since we show it as log in next line.

Reply:
Removed.